### DISCUSSION TITLE
Grab spammer filters from URL

### DISCUSSION THREAD

**Discussion by joaogelado** [ThioJoe/YT-Spammer-Purge#560] [Ideas]
If we used a URL to grab spam filters, we should be able to update the filters to detect current scammers techniques without updating the whole app.

Example: https://tjoe.io/yt-spmr-prg-filters -> List of b64 codes that have the filters

**Comment by Firecul** [comment]
At the moment the app grabs the lists every run, if it's a change more than that I think the program would need to reload anyway to load the new code no?

**Reply by joaogelado** [reply]
yeah but i was thinking for the current filters. others would be ignored if not presented in the current version
